src/test/java/it/pagopa/ecommerce/commons/v1/TransactionTestUtils.java

```diff
@@ -143,10 +143,19 @@ public static TransactionAuthorizationCompleted transactionAuthorizationComplete
         );
     }
 
+    @Nonnull
+    public static TransactionClosedEvent transactionClosedEvent(TransactionClosureData.Outcome outcome) {
+        return new TransactionClosedEvent(
+                TRANSACTION_ID,
+                new TransactionClosureData(outcome)
+        );
+    }
+
     @Nonnull
     public static TransactionClosedEvent transactionClosedEvent() {
         return new TransactionClosedEvent(
-                TRANSACTION_ID
+                TRANSACTION_ID,
+                new TransactionClosureData(TransactionClosureData.Outcome.OK)
         );
     }
```

Comment:
I would probabily add a `@Deprecated`, given that this event is inherently both for OK and KO cases we should be explicit in tests which variant we are testing :)

Reply:
I removed altogether this method since we are performing refactoring on all modules with this commons update to avoid have to perform another refactoring later